.gitignore

```diff
@@ -2,3 +2,4 @@
 .dart_tool
 .pub
 pubspec.lock
+.idea
```

Comment:
Please revert this change. These ignores should be in `~/.gitignore_global`

Reply:
Done